### PR TITLE
Am 516 banner content to come from contentful

### PR DESCRIPTION
**Pull request by sscaife**

### CODE REVIEW

File: src/SFA.DAS.Campaign.UnitTests/Models/WhenBuildingTheBannerPageModelFromApiResponse.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            //Assert
+            actual.Should().BeNull();
+        }
+
+        [Test, RecursiveMoqAutoData]
+        public void Then_The_Banner_Is_Built(CmsContent source, EntryFields linkedPage)
+        {
+            var actual = new BannerPageModel().Build(source);
+
+            //actual.MainContent.Should().NotBeNullOrEmpty();
```
Comment (dashton82) [inline src/SFA.DAS.Campaign.UnitTests/Models/WhenBuildingTheBannerPageModelFromApiResponse.cs:44]: Des this need un-commenting ?

File: src/SFA.DAS.Campaign/ExternalApi/Requests/GetBannerRequest.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Campaign.ExternalApi.Requests
+{
+    public class GetBannerRequest : IGetApiRequest
```
Comment (dashton82) [inline src/SFA.DAS.Campaign/ExternalApi/Requests/GetBannerRequest.cs:5]: Might be a missing test for creating this Request

File: src/SFA.DAS.Campaign/SFA.DAS.Campaign.csproj [hunk 3]
```diff
@@ -5,16 +5,18 @@
...
 
     <ItemGroup>
-      <PackageReference Include="contentful.aspnetcore" Version="6.0.7" />
-      <PackageReference Include="MediatR" Version="6.0.0" />
+      <ProjectReference Include="..\SFA.DAS.SharedOuterApi\SFA.DAS.SharedOuterApi.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-      <Folder Include="Application\Services" />
+      <PackageReference Include="contentful.aspnetcore" Version="6.0.7" />
```
Comment (dashton82) [inline src/SFA.DAS.Campaign/SFA.DAS.Campaign.csproj:18]: Do we need this package reference?

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.Campaign.UnitTests/Models/WhenBuildingTheBannerPageModelFromApiResponse.cs (hunk 1 above)
Comment (sscaife) [reply src/SFA.DAS.Campaign.UnitTests/Models/WhenBuildingTheBannerPageModelFromApiResponse.cs:44]: yeah, it needs an actual assert. doh

File: src/SFA.DAS.Campaign/SFA.DAS.Campaign.csproj (hunk 3 above)
Comment (sscaife) [reply src/SFA.DAS.Campaign/SFA.DAS.Campaign.csproj:18]: yeah it's used to deserialise the response

**Review by dashton82 — APPROVED** [review]

**Review by dashton82 — APPROVED** [review]

**Comment by sony-kotha** [comment]
Testing completed.

**Pull request merged**